Author: brian m. carlson <sandals@crustytoothpaste.net>
Description: Add Debian-specific cons25-debian terminfo entry
 The Debian GNU/kfreeBSD console differs from the stock FreeBSD one in
 respect to the backspace/delete keys, which have been modified to
 behave as in Linux to match the Debian Policy.  This requires a new
 terminfo entry because the backspace key shows erratic behavior with
 the standard cons25 entry.
Bug-Debian: http://bugs.debian.org/607662
Last-Update: 2010-12-29

---
 misc/terminfo.src |    3 +++
 1 file changed, 3 insertions(+)

--- a/misc/terminfo.src
+++ b/misc/terminfo.src
@@ -1774,6 +1774,9 @@
 cons25|ansis|ansi80x25|freebsd console (25-line ansi mode),
 	acsc=-\030.^Y0\333`\004a\260f\370g\361h\261i\025j\331k\277l\332m\300n\305q\304t\303u\264v\301w\302x\263y\363z\362~\371,
 	use=cons25w,
+cons25-debian|freebsd console with debian backspace (25-line ansi mode),
+	kbs=\177, kdch1=\E[3~,
+	use=cons25,
 cons25-m|ansis-mono|ansi80x25-mono|freebsd console (25-line mono ansi mode),
 	colors@, pairs@,
 	bold@, dim@, op@, rmul=\E[m, setab@, setaf@,
